src/test/resources/epub3/files/package-document/mediaoverlays-duration-total-not-sum-warning.opf

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<package xmlns="http://www.idpf.org/2007/opf" version="3.0" unique-identifier="uid"
+    xmlns:dc="http://purl.org/dc/elements/1.1/">
+    <metadata>
+        <dc:title>Title</dc:title>
+        <dc:language>en</dc:language>
+        <dc:identifier id="uid">NOID</dc:identifier>
+        <meta property="dcterms:modified">2019-01-01T12:00:00Z</meta>
+        <!-- Media Overlays Duration Properties -->
+        <meta property="media:duration">10min</meta>
+        <meta refines="#mo001" property="media:duration">5min</meta>
+        <meta refines="#mo002" property="media:duration">0:05:00.200</meta>
```

Comment:
This shouldn't result in a warning. The total has to match the sum of the individual times plus or minus one second to be valid, and this is only off .2s.